crm_sale_marketing/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+================
+Sales Marketing
+================
+
+* This module copies the marketing details while converting opportunity to quotation.
+* Which will help to know number of Sales generated from any particular campaign.
+
+Installation
+============
+
+* No external library is used.
+
+Configuration
+=============
+
+* No Configuration Requrired.
+
+Usage
+=====
+
+* This module will be helpful to trace the Sales generated from the any Marketing source.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/111/9.0
+
+Bug Tracker
```

Comment:
This section has changed. Please apply the same format as in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst